src/Microsoft.ML.FastTree/GamClassification.cs

```diff
@@ -112,29 +112,29 @@ private static bool[] ConvertTargetsToBool(double[] targets)
             return new ValueMapperCalibratedModelParameters<BinaryClassificationGamModelParameters, PlattCalibrator>(Host, predictor, calibrator);
         }
 
-        private protected override ObjectiveFunctionBase CreateObjectiveFunction()
+        protected override ObjectiveFunctionBase CreateObjectiveFunction()
         {
             return new FastTreeBinaryClassificationTrainer.ObjectiveImpl(
                 TrainSet,
                 ConvertTargetsToBool(TrainSet.Targets),
-                Args.LearningRates,
```

Comment:
>Args [](start = 16, length = 4)

Why nor `Options`?

Reply:
I wanted to give a name to this property so that users would easily be able to distinguish to from the class `Options` .  


---
In reply to: [258278906](https://github.com/dotnet/machinelearning/pull/2636#discussion_r258278906) [](ancestors = 258278906)